data/api/client-server/relations.yaml

```diff
@@ -0,0 +1,170 @@
+# Copyright 2022 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+swagger: '2.0'
+info:
+  title: "Matrix Client-Server Relations API"
+  version: "1.0.0"
+host: localhost:8008
+schemes:
+  - https
+  - http
+basePath: /_matrix/client/v1
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  $ref: definitions/security.yaml
+paths:
+  "/rooms/{roomId}/relations/{eventId}/{relType}/{eventType}":
+    get:
+      summary: Get the cgukd events for a given parent event, with optional filter.
+      description: |-
+        Retrieve all of the chikd events for a given parent event, optionally filtering
+        down by relationship type and related event type.
```

Comment:
```suggestion
      summary: Get the child events for a given parent event, with optional filter.
      description: |-
        Retrieve all of the child events for a given parent event, optionally filtering
        down by relationship type and related event type.
```

Reply:
I'm surprised the spellchecker isn't picking these up 😭

Reply:
Maybe you got them so wrong it gave up? 😄